src/prover_disk.hpp

```diff
@@ -254,18 +253,18 @@ class DiskProver {
 
         // This is the checkpoint at the beginning of the park
         uint16_t line_point_size = EntrySizes::CalculateLinePointSize(k);
-        uint8_t* line_point_bin = new uint8_t[line_point_size + 7];
+        auto* line_point_bin = new uint8_t[line_point_size + 7];
```

Comment:
these should probably be `std::unique_ptr<uint8_t[]>` too, (and created with `std::make_unique<uint8_t[]>(line_point_size + 7)`).